packages/fetch-core/src/fetchToken.js

```diff
@@ -7,23 +7,25 @@ const getAccessToken = user => () => {
 
 export const fetchWithToken = (fetch, getAccessTokenInjected) => async (
```

Comment:
this feature is very interesting for projects, is it exposed for use in projects?

Reply:
This function is just a part of the feature. You need a function to get the token (getAccessTokenInjected) and the default function do the sale but, with the function injected.

So for the same thing you just have to use the default function, and so, the fetchToken function of the package